src/dotnet/commands/dotnet-migrate/Program.cs

```diff
@@ -0,0 +1,55 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+
+namespace Microsoft.DotNet.Tools.Migrate
+{
+    public partial class MigrateCommand
+    {
+        public static int Run(string[] args)
+        {
+            DebugHelper.HandleDebugSwitch(ref args);
+
+            CommandLineApplication app = new CommandLineApplication(throwOnUnexpectedArg: false);
+            app.Name = "dotnet migrate";
+            app.FullName = ".NET Migrate Command";
+            app.Description = "Command used to migrate project.json projects to msbuild";
+            app.HandleResponseFiles = true;
+            app.AllowArgumentSeparator = true;
+            app.HelpOption("-h|--help");
+
+            CommandOption template = app.Option("-t|--template-file", "Base MSBuild template to use for migrated app. The default is the project included in dotnet new -t msbuild", CommandOptionType.SingleValue);
+            CommandOption output = app.Option("-o|--output", "Directory to output migrated project to. The default is the project directory", CommandOptionType.SingleValue);
```

Comment:
This option is a little strange, IMO.

If I have a directory `src/MyApp` with the project.json and all the code files in it.  Then I say `dotnet migrate -o C:\temp`, does only the `.csproj` get put in "C:\temp"?  If yes, that seems like it would be an issue becuase the .csproj paths to the code files (like `**/*.cs`) are all relative to the .csproj.  So if migration placed the .csproj in a different directory, it wouldn't work since it couldn't find the code files.